VideoLocker/src/main/java/org/edx/mobile/util/MediaConsentUtils.java

```diff
@@ -14,6 +16,8 @@
  */
 public class MediaConsentUtils {
 
+    @Inject Config config;
```

Comment:
On second thought, why do you even need the `Config` here? I would just use `activity.getString(R.string.platform_name)`